src/input/user_input.rs

```diff
@@ -881,3 +881,17 @@ pub fn view_asterisk(required: bool) -> Html {
         html! {}
     }
 }
+
+pub fn validate_extensions(extensions: &[String]) -> Vec<String> {
```

Comment:
I don't think this method should be public. No one outside Frontary will not need it.

Reply:
I've updated it based on your feedback. Thanks for pointing it out!